bins/revme/src/cmd/statetest/models/mod.rs

```diff
@@ -146,6 +127,30 @@ pub struct Authorization {
     signer: Option<Address>,
 }
 
+impl TestAuthorization {
+    pub fn signature(&self) -> Signature {
+        let v = u64::try_from(self.v).unwrap_or(u64::MAX);
+        let parity = Parity::try_from(v).unwrap();
```

Comment:
You should do `unwrap_or(Parity::Eip155(36))` it will pass as valid Parity by Signature struct but it will be considered invalid in runtime consensus check in Revm as it expects `0/1` (And this is what this test is testing).